src/lib/logger/exceptions.js

```diff
@@ -7,7 +7,7 @@ export const ExceptionCategory = {
   Unexpected: 'unexpected',
 }
 
-export const ExceptionCode = fromPairs(range(1, 10).map(code => [`E${code}`, code]))
+export const ExceptionCode = fromPairs(range(1, 13).map(code => [`E${code}`, code]))
```

Comment:
range doesn't includes the upper bound. so for E1-E-13 codes you have to call range(1, 14). 
probably it would be good to suggest the code as the following to make it more clear:

```suggestion
const MAX_EXCEPTION_CODE = 13

export const ExceptionCode = fromPairs(range(1, MAX_EXCEPTION_CODE + 1).map(code => [`E${code}`, code]))
```